examples/usb/host/src/host.c

```diff
@@ -16,12 +15,9 @@ void shutdown(void)
 
 int main(void)
 {
-  control_version_t version;
+  control_version_t version = 0xFF;
```

Comment:
could you add a comment, maybe a define?

Reply:
I added a define, let me know if more changes are necessary.